From ce7ecf614db32b38cfd99e1756e38387377184db Mon Sep 17 00:00:00 2001
From: Felix Wiedemann <1wiedema@informatik.uni-hamburg.de>
Date: Wed, 20 Apr 2016 14:46:48 +0200
Subject: [PATCH] Pipe stdin to the stdin of the command in the ACI

---
 engine/systemdnspawn/systemdnspawn.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/engine/systemdnspawn/systemdnspawn.go b/engine/systemdnspawn/systemdnspawn.go
index bd000f9..e249c55 100644
--- a/engine/systemdnspawn/systemdnspawn.go
+++ b/engine/systemdnspawn/systemdnspawn.go
@@ -75,6 +75,7 @@ func (e Engine) Run(command string, args []string, environment types.Environment
 	nspawncmd = append(nspawncmd, args...)
 
 	execCmd := exec.Command(nspawncmd[0], nspawncmd[1:]...)
+	execCmd.Stdin = os.Stdin
 	execCmd.Stdout = os.Stdout
 	execCmd.Stderr = os.Stderr
 	execCmd.Env = []string{"SYSTEMD_LOG_LEVEL=err"}
-- 
2.1.4

