From cea57d951d6cf494d7fd0e85ca0340ab37262410 Mon Sep 17 00:00:00 2001
From: Nils Rokita <0rokita@informatik.uni-hamburg.de>
Date: Tue, 7 Mar 2017 13:21:52 +0100
Subject: [PATCH] hotfix image upload problem

---
 app/controllers/uploads_controller.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/controllers/uploads_controller.rb b/app/controllers/uploads_controller.rb
index d33adb32e..7d0740a2f 100644
--- a/app/controllers/uploads_controller.rb
+++ b/app/controllers/uploads_controller.rb
@@ -22,10 +22,10 @@ class UploadsController < ApplicationController
       data = create_upload(type, file, url)
       render json: data.as_json
     else
-      Scheduler::Defer.later("Create Upload") do
+      #Scheduler::Defer.later("Create Upload") do
         data = create_upload(type, file, url)
         MessageBus.publish("/uploads/#{type}", data.as_json, client_ids: [client_id])
-      end
+      #end
       render json: success_json
     end
   end
-- 
2.11.0

