diff --git a/SoObjects/SOGo/LDAPSource.m b/SoObjects/SOGo/LDAPSource.m
index 6de1036d1..3968f4f6f 100644
--- a/SoObjects/SOGo/LDAPSource.m
+++ b/SoObjects/SOGo/LDAPSource.m
@@ -2321,7 +2321,7 @@ _makeLDAPChanges (NGLdapConnection *ldapConnection,
   NSMutableArray *dns, *uids, *userLogins;
   NSString *dn, *login;
   SOGoUserManager *um;
-  NSDictionary *d, *contactInfos;
+  NSDictionary *d;
   SOGoUser *user;
   NSArray *o, *subusers, *logins;
   NSAutoreleasePool *pool;
@@ -2381,21 +2381,7 @@ _makeLDAPChanges (NGLdapConnection *ldapConnection,
               user = [SOGoUser userWithLogin: login  roles: nil];
               if (user)
                 {
-                  if (!_disableSubgroups) {
-                    contactInfos = [self lookupContactEntryWithUIDorEmail: login inDomain: nil];
-                    if ([contactInfos objectForKey: @"isGroup"])
-                    {
-                      subusers = [self membersForGroupWithUID: login];
-                      [members addObjectsFromArray: subusers];
-                    }
-                    else
-                    {
-                      [members addObject: user];
-                    }
-                  } else {
                     [members addObject: user];
-                  }
-                  
                 }
               [pool release];
             }
@@ -2415,20 +2401,7 @@ _makeLDAPChanges (NGLdapConnection *ldapConnection,
               user = [SOGoUser userWithLogin: login  roles: nil];
               if (user)
                 {
-                  if (!_disableSubgroups) {
-                    contactInfos = [self lookupContactEntryWithUIDorEmail: login inDomain: nil];
-                    if ([contactInfos objectForKey: @"isGroup"])
-                    {
-                      subusers = [self membersForGroupWithUID: login];
-                      [members addObjectsFromArray: subusers];
-                    }
-                    else
-                    {
-                      [members addObject: user];
-                    }
-                  } else {
-                    [members addObject: user];
-                  }
+                  [members addObject: user];
                 }
               [pool release];
             }
